src/panels/TimePanel/TimeInput/TimeInput.tsx

```diff
@@ -354,10 +351,10 @@ export function TimeInput() {
               }}
               variant={'filled'}
             >
-              Set
+              {t('lock-time.button-labels.set')}
             </Button>
             <Button variant={'default'} onClick={() => setUseLock(false)}>
-              Cancel
+              {t('common:cancel')}
```

Comment:
For some reason, this string is no longer capitalized. It should be

![image](https://github.com/user-attachments/assets/53322f1b-e147-4b43-9aff-3ce364d74647)

Reply:
Actually, the issue is that the key isn't found
![image](https://github.com/user-attachments/assets/c27032d1-fae9-445b-b353-2b30059a2977)

The issue is the `keyPrefix` being `time-input`, and it seems like this prefix is applied to both the included namespaces. 

One that I used to include the common namespace in #144 is like this: 
```
  const { t: tCommon } = useTranslation('common');
  const { t } = useTranslation('panel-keybinds', { keyPrefix: 'keybind-info' });
```
maybe doing it like that would be enough?  Or do we prefer to do a namespaced solution?